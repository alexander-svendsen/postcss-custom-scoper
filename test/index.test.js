import { run, runAndExpectResult, compareFixtureWithSnapshot } from './helpers';

it('Basic functionality', () => {
    compareFixtureWithSnapshot('base.css');
});

it('', () => {
    const { css } = run('.foo { color: yellow }');
    expect(css).toEqual('blee { color: yellow }');
});
