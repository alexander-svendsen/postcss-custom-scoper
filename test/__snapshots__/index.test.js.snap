// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Basic functionality 1`] = `
"
blee { 
  color: yellow
}
"
`;
